### ISSUE TITLE
Self-loop does not show up when drawing the graph

### ISSUE THREAD

**Issue by djzts** [networkx/networkx#4865]
### Current Behavior

I was trying to draw some self-looped nodes and edges. But the output below does not show any self-loop edges as it should be.

![image](https://user-images.githubusercontent.com/25216627/120393640-de774080-c2f7-11eb-95bd-5183f585a81d.png)

### Expected Behavior

The source web page is 
https://networkx.org/documentation/latest/auto_examples/drawing/plot_selfloops.html

### Environment

My platform: Jupyter Notebook.
Python version: Python3.8
NetworkX version: Version: 2.5.1

**Comment by rossbar** [comment]
Self-loops are not included in `draw_networkx_edges` in networkx version 2.5 (see the comment on line 9).

The good news is this feature has been added in networkx 2.6, which will be released soon. You can try it out now by installing the release candidate, e.g. `pip install --upgrade --pre networkx`.

**Comment by rossbar** [comment]
Self-loops are not included in `draw_networkx_edges` in networkx version 2.5 (see the comment on line 9).

The good news is this feature has been added in networkx 2.6, which will be released soon. You can try it out now by installing the release candidate, e.g. `pip install --upgrade --pre networkx`.

**Comment by djzts** [comment]
Thank you for the clarification. 
There is a follow-up Question. When I was drawing the label on a curved line in Digraph. The label does not align with the curved arrow line like below. Is it got fixed in version 2.6?
![image](https://user-images.githubusercontent.com/25216627/120553952-bef71b00-c3be-11eb-9ca7-22d0a5d4cbee.png)

**Comment by rossbar** [comment]
No, AFAICT having the labels follow the edge arcs is not currently supported. If that's a feature you'd like to discuss, or you have any other unrelated drawing questions, I'd recommend that you open a new issue so that it doesn't get buried in this thread on an unrelated topic.
